sklearn/impute/tests/test_impute.py

```diff
@@ -971,6 +971,59 @@ def test_iterative_imputer_catch_warning():
     assert not np.any(np.isnan(X_fill))
 
 
+@pytest.mark.parametrize("min_value, max_value",
+                         [(0, 100), (None, None), (-np.inf, np.inf),
+                          ((-5, 5, 10), (100, 200, 300)),
+                          ((-5, None, 10), (100, 200, None))])
+def test_iterative_imputer_min_max_completion(min_value, max_value):
```

Comment:
an additional test is to check that passing an array is equivalent to scalar.
additionally, passing array of `np.inf` and `-np.inf` is equivalent to `None`

Reply:
Those two tests are already part of the cases I have put in there. I have added comments to explain each test case, which should make it clearer.